packages/vuetify/src/composables/display.ts

```diff
@@ -0,0 +1,202 @@
+// Utilities
+import { inject, reactive, ref, toRefs, watchEffect } from 'vue'
+import { mergeDeep } from '@/util'
+
+// Globals
+import { IN_BROWSER, SUPPORTS_INTERSECTION, SUPPORTS_TOUCH } from '@/util/globals'
+
+// Types
+import type { InjectionKey, ToRefs } from 'vue'
+
+export type DisplayBreakpoint = keyof DisplayThresholds
+
+export interface DisplayThresholds {
+  xs: number
```

Comment:
properties here (xs/sm/md/lg) don't match properties in sass (sm/md/lg/xl) https://github.com/vuetifyjs/vuetify/blob/b72f91e962421e1fe2d3795701e5667cee582a6c/packages/vuetify/src/styles/settings/_variables.scss#L92-L96 and afair that was one of the things that we were going to unify